### DISCUSSION TITLE
Check Script

### DISCUSSION THREAD

**Discussion by HitMannCU** [RandomNinjaAtk/arr-scripts#109] [Q&A] [answered]
Hello there!. Is there a way to actually check that the scripts are really working? Sorry if it is a dumb question: just need to know what to check in the logs to know is or isn't working, Thanks in advance!

**Comment by RandomNinjaAtk** [comment, marked as answer]
https://github.com/RandomNinjaAtk/arr-scripts/discussions/103#discussioncomment-7411121

**Comment by HitMannCU** [comment]
the thing is that i should be doing something wrong, because the only way i see any scripts logs is when i run them manually from within the container. If i restart the host, they don't start
